bundles/org.openhab.binding.comfoair/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,1177 @@
+		<description>Inlet air temperature (inside)</description>
+		<category>Temperature</category>
+		<state readOnly="true" pattern="%.1f %unit%"/>
+	</channel-type>
+
+	<channel-type id="indoor_temperature_out">
+		<item-type>Number:Temperature</item-type>
+		<label>Indoor Temperature Outgoing</label>
+		<description>Uptake air temperature (inside)</description>
+		<category>Temperature</category>
+		<state readOnly="true" pattern="%.1f %unit%"/>
+	</channel-type>
+
+	<channel-type id="is_T1_sensor" advanced="true">
+		<item-type>Switch</item-type>
+		<label>Sensor T1 Available</label>
+		<description>Availability of temperature sensor T1 (outdoor in)</description>
+		<category>Temperature</category>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="is_T2_sensor" advanced="true">
+		<item-type>Switch</item-type>
+		<label>Sensor T2 Available</label>
+		<description>Availability of temperature sensor T2 (indoor in)</description>
+		<category>Temperature</category>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="is_T3_sensor" advanced="true">
+		<item-type>Switch</item-type>
+		<label>Sensor T3 Available</label>
+		<description>Availability of temperature sensor T3 (indoor out)</description>
+		<category>Temperature</category>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="is_T4_sensor" advanced="true">
+		<item-type>Switch</item-type>
+		<label>Sensor T4 Available</label>
+		<description>Availability of temperature sensor T4 (outdoor out)</description>
+		<category>Temperature</category>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="is_EWT_sensor" advanced="true">
+		<item-type>Switch</item-type>
+		<label>EWT Sensor Available</label>
+		<description>Availability of EWT temperature sensor</description>
+		<category>Temperature</category>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="is_heater_sensor" advanced="true">
+		<item-type>Switch</item-type>
+		<label>Heater Sensor Available</label>
+		<description>Availability of heater temperature sensor</description>
+		<category>Temperature</category>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="is_cookerhood_sensor" advanced="true">
+		<item-type>Switch</item-type>
+		<label>Cookerhood Sensor Available</label>
+		<description>Availability of cookerhood temperature sensor</description>
+		<category>Temperature</category>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="chimney_state" advanced="true">
+		<item-type>Switch</item-type>
+		<label>Chimney Control State</label>
+		<description>State of the chimney control</description>
+		<category>Switch</category>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="bypass_state" advanced="true">
+		<item-type>Switch</item-type>
+		<label>Bypass State</label>
+		<description>State of the bypass (ON = open / OFF = closed)</description>
+		<category>Switch</category>
+		<state readOnly="true"/>
```

Comment:
Did you take a look at the item-type `Contact`? It has the two states OPEN and CLOSED.

Reply:
Yes thought of that too already. However, I'm not quite sure if this really fits (better). While it would fit in terms of the state, it is not really a contact but flap that is opening/closing. Additionally, this is a self-regulating function of the device, so it could also be seen as ENABLED/DISABLED. Therefore, and to be consistent with the other device option states, I would prefer to keep this as is.